### ISSUE TITLE
No statusText on Chrome

### ISSUE THREAD

**Issue by FlandreDaisuki** [axios/axios#1501]
#### Summary

The response should include `status: 200` and `statusText: "OK"` but Chrome returns the empty string in `statusText` property.
Firefox is OK.

##### Steps to reproduce:
1. copy `dist/axios.min.js` to console
2. paste following code to Chrome console
    ```
    (async () => {
      const res = await axios.get(location.href);
      console.log('status', res.status);
      console.log('statusText', `"${res.statusText}"`);
    })();
    ```
#### Context

- axios version: 
   - v0.18.0
   - v0.17.1
- Environment: 
   - Google Chrome (66.0.3359.117 64bit)
   - Google Chrome (66.0.3359.139 64bit)

**Comment by lonnylot** [comment]
I have the same experience with Safari. 

axios version 0.18.0
Safari Version 11.1 (13605.1.33.1.4)

**Comment by JonasJonny** [comment]
@mzabriskie @nickuraltsev 
I lost two days with this "issue" (Axios version 0.16.2) cause it is a feature and not a bug :D. 

My app is using `statusText: "OK"` but in Chrome 49+ is `statusText: ""` empty. 
Actually older versions (<= v.48 `"OK"`) looks good. Opera 36+ empty, Safari 10.1.1 empty.

Important to know the problem is only on `HTTPS`. Http is working fine everywhere.

_After deep searching it could be all related to:_
- **https://github.com/whatwg/fetch/issues/599**
- https://stackoverflow.com/questions/41632077/why-is-the-statustext-of-my-xhr-empty
- https://stackoverflow.com/questions/42401795/with-http-2-only-xmlhttprequest-responses-statustext-property-seems-to-be-us

**Comment by MetaiR** [comment]
any solution?

**Comment by JonasJonny** [comment]
@MetaiR I started to use `response.status === 200`.

**Comment by JonasJonny** [comment]
@MetaiR I dropped "OK" and started to use response.status === 200.

**Comment by huruji** [comment]
is it a bug of chrome?

**Comment by JonasJonny** [comment]
> is it a bug of chrome?

Not only.

> Chrome 49+ empty, Opera 36+ empty, Safari 10.1.1 empty.

Drop `response.statusText` and use `response.status` as solution.

**Comment by narrowizard** [comment]
> @MetaiR I dropped "OK" and started to use response.status === 200.

`response.status >= 200 && response.status < 300` is more reliable.

**Comment by chinesedfan** [comment]
Closed due to not an issue of axios. Feel free to remind me if someone found something that requires axios to do.

**Comment by narrowizard** [comment]
Definitionally, it is not a bug. see https://stackoverflow.com/a/41637669/5760975   
>    HTTP/2 does not define a way to carry the version or reason phrase
   that is included in an HTTP/1.1 status line.

+ reference: [RFC 7540](https://tools.ietf.org/html/rfc7540#section-8.1.2.4)
